### ISSUE TITLE
Studied Target + Standard action causes loss of standard action

### ISSUE THREAD

**Issue by Valdacil** [cabarius/ToyBox#69]
Playing Inquisitor [Sanctified Slayer] which gains the Studied Target feature as a Move action at level 1.  Playing in Turn Based Mode.  Played without any modes and Studied Target works properly as a Move action.  Added Toy Box and restarted a new game/character.  When I use Studied Target it properly shows that it used up the Move action and I have 5ft Step and Standard action available.  If I attack an adjacent foe (don't need to move) I receive the voice confirmation that he is attacking, the standard action is consumed, but no attack occurs.  If I disable Toy Box then close and restart the game with Toy Box disabled then Studied Target + Standard action works properly again.

Steps to Reproduce:

1. Create an Inquisitor [Sanctified Slayer] and start the game
2. Set to Turn Based combat
3. Save right before a combat
4. Get into combat and get foe adjacent to PC
5. Use Studied Target and not that 5ft Step, Standard Action, and Swift Action are still available.
6. Hover over the attack and note that it shows that the attack will consume the Standard Action
7. Execute attack and note that Standard action is consumed but no attack occurs
8. Disable Toy Box, close game, restart game and load save
9. Repeat steps 4-7 except note that Step 7 performs the attack

**Comment by Valdacil** [comment]
I'm pretty sure this is affecting all Move Equivalent actions, not just Studied Target.  I'm playing a Paladin [Warrior of the Holy Light] now that doesn't have Studied Target, but I just drank a potion as a Move Equivalent action and it showed I still had a Standard Action left.  I attacked and the attack was lost but it consumed the Standard Action.

**Comment by Valdacil** [comment]
There have no also been reports of this happening with Kineticist which must Gather Power (Move action) before executing a Blast (Standard action).  This bug completely breaks the ability of that class to function properly.

**Comment by BrutaleBent** [comment]
Yeah, this is a gamebreaking bug for me - at first I thought it was just the base game being wonky still.

**Comment by Valdacil** [comment]
1.2.9.1 seems to have fixed the problem.

**Comment by Amariithynar** [comment]
> 1.2.9.1 seems to have fixed the problem.

If this is true, would recommend as issue opener that you close your issue!
